frontend/ui/elements/screensaver_menu.lua

```diff
@@ -26,114 +26,144 @@ local function genMenuItem(text, setting, value, enabled_func, separator)
         separator = separator,
     }
 end
-
 return {
-    genMenuItem(_("Use last book's cover as screensaver"), "screensaver_type", "cover", hasLastFile),
-    genMenuItem(_("Use book status as screensaver"), "screensaver_type", "bookstatus", hasLastFile),
-    genMenuItem(_("Use random image from folder as screensaver"), "screensaver_type", "random_image"),
-    genMenuItem(_("Use document cover as screensaver"), "screensaver_type", "document_cover"),
-    genMenuItem(_("Use image as screensaver"), "screensaver_type", "image_file"),
-    genMenuItem(_("Use reading progress as screensaver"), "screensaver_type", "readingprogress", isReaderProgressEnabled),
-    genMenuItem(_("Leave screen as-is"), "screensaver_type", "disable", nil, true),
-    -- separator
-    {
-        text = _("Add message to screensaver"),
-        checked_func = function()
-            return G_reader_settings:isTrue("screensaver_show_message")
-        end,
-        callback = function()
-            G_reader_settings:toggle("screensaver_show_message")
-        end,
-        separator = true,
-    },
-    -- separator
     {
-        text = _("Settings"),
+        text = _("Screensaver"),
         sub_item_table = {
+            genMenuItem(_("Show book cover on sleep screen"), "screensaver_type", "cover", hasLastFile),
+            genMenuItem(_("Show custom image on sleep screen"), "screensaver_type", "image_file"),
+            genMenuItem(_("Shuffle images on sleep screen"), "screensaver_type", "random_image"),
+            genMenuItem(_("Show document cover on sleep screen"), "screensaver_type", "document_cover"),
```

Comment:
I'm not sure what was/is the different between `Show book cover` and `Show document cover` , but the previous wording `Use last book's cover` vs `Use document cover` made it look/read more different :) 

Also feels a bit odd to have these 2 "image" related options in between these 2 book/document options.

Reply:
> Also feels a bit odd to have these 2 "image" related options in between these 2 book/document options.

I agree, I think they should be just one option where you select a folder and it will shuffle through the images there, whether they are 1 or 1000. 

> I'm not sure what was/is the different between `Show book cover` and `Show document cover`

someone also commented about merging those two, but alas I also don't quite know what the document one is all about.

Reply:
Last book's cover vs document cover doesn't seem overly unclear. Last (book's) cover is whatever you're reading, vs a static (document) cover that'll be shown always and forever. NB I'm not saying that's what it is; if I'm wrong it's not clear but if I'm right it's at least somewhat clear. ;-) Also I'm explicitly not advocating to keep it! But book cover vs document cover is meaningless.

Reply:
I will mark this as resolved as @Frenzie will fix it with his upcoming PR that unifies documents and images...